templates/receipt/templates/main.tpl

```diff
@@ -0,0 +1,33 @@
+{{define "simple-paragraph"}}
```

Comment:
You define a reusable `simple-paragraph` sub-template here but only use it in the `table-cell-paragraph` sub-template once. Might as well define the paragraph there in this case. The `simple-paragraph` sub-template is not necessary.